Software/src/wizard/ConfigureUdpDevicePage.ui

```diff
@@ -102,6 +102,9 @@
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="sbTimeout">
+         <property name="toolTip">
+          <string>Setting this to 255 means the device will keep the colors set by Prismatik. For other values, the device will return to its previous state when Prismatik is closed.</string>
```

Comment:
> when Prismatik is closed

that's just one of the cases

I'd detail even more, something like:
> Timeout in seconds after which the device returns to normal mode if no data is sent.
Setting this to 255 disables the timeout and the device will keep the last received colors after no more data is sent.
It's recommended to not use "Send data only if colors changed" setting with a short timeout value.

Reply:
Done :)